### DISCUSSION TITLE
why I can't add work node more than 16?

### DISCUSSION THREAD

**Discussion by linxd** [citusdata/citus#7206] [General]
I have a citus cluster with 15 node, all are use refference table.
but I can't do :   SELECT * from citus_add_node('100.xx', 5432), got : 

 WARNING:  could not establish connection after 30000 ms

错误： connection to the remote node 100.xxx:5432 failed 

ERROR:  connection to the remote node 100.xxx:5432 failed
SQL 状态: 08006

why?

**Comment by linxd** [comment]
I can add work node in other databse in the same server with same work node ip, but only few work node.

**Comment by hanefi** [comment]
There is no such limitation on the number of nodes. Can you verify that the new nodes are accessible by all the nodes in your cluster?
